test/data/css/tests.js

```diff
@@ -1800,6 +1800,48 @@ exports.test_data = {
         ]
       }]
     }, {
+      name: "space_in_paren true",
+      description: "",
+      options: [
+        { name: "space_in_paren", value: "true" }
+      ],
+      tests: [{
+        fragment: true,
+        input: [
+          'a {',
+          'width: min(100%,100vw);',
```

Comment:
Please add another test where the input has multiple spaces in inside the paren, example: `'width: min(. 
  100%. ,  100vw.   );',`.  In the case the extra spaces should be removed.

Reply:
Thanks for the feedback! Will add them in the coming week.